osu.Game/Skinning/SkinManager.cs

```diff
@@ -30,6 +30,13 @@
 
 namespace osu.Game.Skinning
 {
+    /// <summary>
+    /// Handles the storage and retrieval of <see cref="Skin"/>s.
+    /// </summary>
+    /// <remarks>
+    /// This is also exposed and cached as <see cref="ISkinSource"/> on a game-wide level for general components across the game.
+    /// Lookups from gameplay components are instead covered by <see cref="RulesetSkinProvidingContainer"/>, and are never hit here.
```

Comment:
```suggestion
    /// Lookups from gameplay components are instead covered by <see cref="RulesetSkinProvidingContainer"/>, and never hit here.
```

Typo, but best just remove this now?

Reply:
Remove the whole line? it will still not reach `SkinManager` though since `RulesetSkinProvidingContainer` disallows falling back?